db/migrate/20190604090631_create_saved_urls.rb

```diff
@@ -0,0 +1,12 @@
+class CreateSavedUrls < ActiveRecord::Migration[5.0]
+  def change
+    create_table :saved_urls do |t|
+      t.bigint  "refers_to_id", :type => :bigint
+      t.string  "refers_to_type"
```

Comment:
`t.references :refers_to, polymorphic: true, index: true`

should create the `refers_to_id` and ..._type` and the index